util/find-doc-nits

```diff
@@ -45,11 +46,12 @@ our($opt_c);
 sub help {
     print <<EOF;
 Find small errors (nits) in documentation.  Options:
-    -c List undocumented commands and options
+    -c List undocumented commands and options and not implemented options
```

Comment:
_List undocumented commands, undocumented options and unimplemented options._

Reply:
Done.